packages/http-client/package.json

```diff
@@ -1,37 +1,45 @@
 {
   "name": "@actions/http-client",
-  "version": "1.0.11",
+  "version": "2.0.0",
```

Comment:
can we add/update the releases.md with information on why we have published a new major version, update the readme, and let users know why they should use it?